backend/main.go

```diff
@@ -1,22 +1,18 @@
 package main
 
 import (
-	"log"
-	"net/http"
+	"flag"
 
-	"github.com/gorilla/mux"
-	"github.com/ibeckermayer/teleport-interview/backend/internal/handlers"
+	"github.com/ibeckermayer/teleport-interview/backend/internal/server"
 )
 
 func main() {
-	router := mux.NewRouter()
+	port := flag.Int("port", 8000, "Port to serve the app from")
+	certFilePath := flag.String("cert", "../certs/localhost.crt", "Relative path to a valid SSL cert")
+	keyFilePath := flag.String("key", "../certs/localhost.key", "Relative path to the cert's private key")
```

Comment:
I don't think this works, if I am not mistaken you have to call `flag.Parse()` before using the variables.